client/templates/orderDetails/orderDetails.html

```diff
@@ -1,8 +1,8 @@
 <template name='orderDetails'>
-  {{> afNavbar}}
-  {{#if Template.subscriptionsReady}}
+{{#if Template.subscriptionsReady}}
   {{#with order}}
     <div class='container-fluid'>
+    <!-- I don't think does anything -->
```

Comment:
That was me wondering why there was an if/esle but it's bases on size and hiding it down.